src/ripple/rpc/impl/Tuning.h

```diff
@@ -46,7 +46,7 @@ static LimitRange constexpr accountObjects = {10, 200, 400};
 static LimitRange constexpr accountOffers = {10, 200, 400};
 
 /** Limits for the book_offers command. */
-static LimitRange constexpr bookOffers = {0, 300, 400};
+static LimitRange constexpr bookOffers = {0, 60, 100};
```

Comment:
What happens if you specify a `limit=200` for example? To my understanding, that would have been allowed in the old API but it's too large in the new API. Depending on the behavior, this may or may not be a breaking change that requires clients to be rewritten:

- If the limit is quietly clamped to the new max limit of 100, that's OK. Clients already need to assume that the API may return fewer than their specified limit.
- If the command now returns an error like rpcINVALID_PARAMS, that's a problematic breaking change.

Reply:
Limits are quietly clamped.

Reply:
These limits also are only applied to non-admin clients